### PR TITLE
ci-ipsec-e2e: Misc refactor + more keys

### PR DESCRIPTION
**Pull request by brb**

### CODE REVIEW

File: .github/workflows/conformance-ipsec-e2e.yaml [hunk 1]
```diff
@@ -113,6 +119,8 @@ jobs:
             encryption: 'ipsec'
             encryption-node: 'false'
             endpoint-routes: 'true'
+            key-one: 'gcm(aes)'
+            key-two: 'gcm(aes)'
```
Comment (pchaigno) [inline .github/workflows/conformance-ipsec-e2e.yaml:123]: Maybe worth testing the reverse, cbc -> gcm?

**Review by pchaigno — APPROVED** [review]

File: .github/workflows/conformance-ipsec-e2e.yaml (hunk 1 above)
Comment (brb) [reply .github/workflows/conformance-ipsec-e2e.yaml:123]: Good point, let me add it!

**Review by jschwinger233 — APPROVED** [review]
:candy:

**Comment by julianwiedmann** [comment]
I believe this depends on https://github.com/cilium/cilium/pull/29704, which we won't backport to v1.13. So let's drop the label here as well.

**Pull request merged**